index.html

```diff
@@ -4,6 +4,7 @@
   <link href="main.css" rel="stylesheet">
 </head>
   <body>
+    <form id="survey-form"></form>  
```

Comment:
Let's move this form down below the title.  We want the title to be at the top of the page.

Reply:
dammit all to hell